metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -376,46 +377,58 @@
                                                            (su/find-one conn (select '[*]
                                                                                      (from table)
                                                                                      (where `(= :concept-id ~concept-id))
-                                                                                     (order-by (desc :revision-id)))))))))]
-     (when (not (= "efs-off" (efs-config/efs-toggle)))
+                                                                                     (order-by (desc :revision-id)))))))))
+         dynamo-concept-get (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
+                              (util/time-execution
+                               ()))]
+     (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
        (info "Runtime of EFS get-concept: " (first efs-concept-get) " ms.\nValue gotten from EFS get-concept: " (second efs-concept-get)))
-     (when (not (= "efs-only" (efs-config/efs-toggle)))
+     (when (not= "dynamo-only" (dynamo-config/dynamo-toggle))
        (info "Runtime of Oracle get-concept: " (first oracle-concept-get) " ms."))
+     (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
```

Comment:
The when on line 388 is the same as the when on line 384. Can you put the two info's together or is one of the whens wrong?